src/Arcus.Observability.Telemetry.Serilog/KubernetesEnricher.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using GuardNet;
+using Serilog.Core;
+using Serilog.Events;
+
+namespace Arcus.Observability.Telemetry.Serilog
+{
+    /// <summary>
+    /// Enrichment on log events that automatically adds Kubernetes information from the environment.
+    /// </summary>
+    public class KubernetesEnricher : ILogEventEnricher
+    {
+        private const string NodeNameVariable = "KUBERNETES_NODE_NAME",
+                             PodNameVariable = "KUBERNETES_POD_NAME",
+                             NamespaceVariable = "KUBERNETES_NAMESPACE";
+
+        private const string NodeNameProperty = "NodeName",
+                             PodNameProperty = "PodName",
+                             NamespaceProperty = "Namespace";
+
+        private readonly IDictionary<string, string> _kubernetesEnvironmentToLogProperty;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="KubernetesEnricher"/> class.
+        /// </summary>
+        public KubernetesEnricher() : this(NodeNameVariable, PodNameVariable, NamespaceVariable)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="KubernetesEnricher"/> class.
+        /// </summary>
+        /// <param name="nodeNameVariableName">The name of the environment variable that specifies the Kubernetes node name.</param>
+        /// <param name="podNameVariableName">The name of the environment variable that specifies the Kubernetes pod name.</param>
+        /// <param name="namespaceVariableName">The name of the environment variable that specifies the Kubernetes namespace.</param>
+        public KubernetesEnricher(string nodeNameVariableName, string podNameVariableName, string namespaceVariableName)
+            : this(new Dictionary<string, string>
```

Comment:
We can get rid of this approach, we don't need this complexity for now imo, can just be stored in a field